src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/Marshalling/MarshallingGenerator.cs

```diff
@@ -56,7 +56,11 @@ public enum ValueBoundaryBehavior
         /// <summary>
         /// The address of the native identifier provided by <see cref="StubCodeContext.GetIdentifiers(TypePositionInfo)"/> should be passed by value.
         /// </summary>
-        AddressOfNativeIdentifier
+        AddressOfNativeIdentifier,
```

Comment:
```suggestion
        AddressOfNativeIdentifier,
        
```

Reply:
Same above? Missing line between value and comment.

Reply:
Added throughout.